11-Backbonejs/03-Backbone-Models.md

```diff
@@ -67,44 +68,52 @@ console.log("The person is named " + ada.get("name"));
 ```
 
 Results in:
-
 ```console
 Person has been Created
 The person is named Ada
 ```
 
-You can also pass in attributes to a model at instantiation via a JSON object in the parameters to new.
+You can also pass in **attributes** to a model at instantiation via a JSON object in the parameters to new.
 
 ```javascript
 var babbage = new Person({name: "Charles Babbage", age: 65, skills: "Hardware Design, Mathematics, Flower Arrangement."});
 ```
 
+These attributes do not necessarily need to have default values specified, or really be specified in any way. In the example above, we've provided two of the fields that we decided to set defaults for, and added another, `skills`.
+
 ## Get, Set & Unset
 
-Once you have an instance of your Model you can use get & set methods to set it's attributes.
+Once you have an instance of your Model you can use `get` & `set` methods to set it's attributes. You may pass individual keys with values, or you may pass an object containing any and all data that you might want to change.
```

Comment:
- "its" instead of "it's".
- We should probably refer to `set` and `get` as functions.
- We should say "For the `set` function you may pass two arguments, an attribute name and the attribute's value. Or you may pass an object containing multiple attributes to be set on the model."